Wikipedia/Localizations/ps.lproj/Localizable.strings

```diff
@@ -182,3 +183,5 @@
 "page-similar-titles" = "ورته مخونه";
 "captcha-reload" = "يو بل کپچا ښکاره کول";
 "captcha-prompt" = "له پورته انځور څخه کپچا متن ورکړئ";
+"home-more-like-footer" = "د $1 په څېر نور";
```

Comment:
@niedzielski is it, I wonder, just a single translator who is not aware these are never to be changed? Or do we need to add some sort of note to the TWN projects for this?